narwhals/stable/v1/__init__.py

```diff
@@ -948,6 +949,25 @@ def wrapper(*args: Any, **kwargs: Any) -> Any:
         return decorator(func)
 
 
+def to_py_scalar(scalar: Any) -> Any:
+    """If a scalar is not Python native, tries to convert it to Python native.
+
+    Examples:
+        >>> import narwhals.stable.v1 as nw
+        >>> import pandas as pd
+        >>> df = nw.from_native(pd.DataFrame({"a": [1, 2, 3]}))
+        >>> nw.to_py_scalar(df["a"].item(0))
+        np.int64(1)
```

Comment:
Should we do the same here (numpy arrays of length one)as we do for cupy? I think it would be more consistent

Reply:
Yeah true, I was a bit unsure since numpy scalars are _basically_ like Python native AFAIK (eg `np.int64(1) == 1`)

But why not? 👌🙂